src/cli/templates/create/_._editorconfig

```diff
@@ -0,0 +1,18 @@
+# EditorConfig: http://EditorConfig.org
+
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[{package.json,package-lock.json,*.yml}]
+indent_style = space
+indent_size = 2
+
+[Makefile]
+indent_style = tab
```

Comment:
Should also probably have `trim_trailing_whitespace = false` as it has its legal usage in MD

Reply:
Good idea!

Reply:
Done.

Reply:
Hm, you say it's done but I don't see it in GH diff view.